hound_test.rb

```diff
@@ -0,0 +1,2 @@
+puts 'something'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.